manifests/init.pp

```diff
@@ -203,6 +205,7 @@
   $user               = $::jenkins::params::user,
   $manage_group       = $::jenkins::params::manage_group,
   $group              = $::jenkins::params::group,
+  $config_template    = undef,
```

Comment:
This should be renamed since the param is not a path to a template but I'm unsure as to what the best name would be.  Maybe `global_config`?